forms-flow-api/src/formsflow_api/services/factory/keycloak_group_service.py

```diff
@@ -174,16 +177,25 @@ def search_group(self, search, data):
         )
         return search_list
 
-    def format_response(self, data):
+    @user_context
+    def format_response(self, data, **kwargs):
         """Format group response."""
+        user: UserContext = kwargs["user"]
+        tenant_key = user.tenant_key
         data["description"] = ""
+        data["permissions"] = []
         data["name"] = data.get("path")
         if data.get("attributes") != {}:  # Reaarange description
             data["description"] = (
                 data["attributes"]["description"][0]
                 if data["attributes"].get("description")
                 else ""
             )
+        if data.get("clientRoles") != {}:  # Reaarange permissions
+            client_role = f"{tenant_key}-forms-flow-web" if tenant_key else "forms-flow-web"
```

Comment:
Can you use `KEYCLOAK_WEB_CLIENT_ID` from environment instead of `forms-flow-web` if not multi tenant setup ?

Reply:
Done